modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -146,12 +146,14 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
                 }
             } else {
                 // allow only to view own site data
-                $site =& Site::singleton($user->getData('CenterID'));
-                if ($site->isStudySite()) {
-                    $list_of_sites = array();
-                    $user_site     =$user->getData('Site');
-                    $list_of_sites[$user->getData('CenterID')] = $user_site;
+                $site_arr = $user->getData('CenterIDs');
+                foreach ($site_arr as $key => $val) {
+                    $site[$key] = &Site::singleton($val);
+                    if ($site[$key]->isStudySite()) {
+                        $list_of_sites[$val] = $site[$key]->getCenterName();
+                    }
```

Comment:
@driusan 

agree. But this occurs in many modules. Should I do this now for 17.1 for all modules?
The other thing I noticed is that I need to define: ` $list_of_sites =  array() `  before the `else` loop as otherwise, if the user is at NO study sites, the next line (Line 156) will probably return a 500 error?
this also applies to many modules.